vmdb/app/views/layouts/_show_statistics.html.haml

```diff
@@ -0,0 +1,86 @@
+      %th
+        = Dictionary::gettext('avg_write_size', :type => :column, :notfound => :titleize)
+      %th
+        = Dictionary::gettext('pct_read', :type => :column, :notfound => :titleize)
+      %th
+        = Dictionary::gettext('pct_write', :type => :column, :notfound => :titleize)
+      %th
+        = Dictionary::gettext('pct_hit', :type => :column, :notfound => :titleize)
+      %th
+        = Dictionary::gettext('utilization', :type => :column, :notfound => :titleize)
+      %th
+        = Dictionary::gettext('response_time_sec', :type => :column, :notfound => :titleize)
+      %th
+        = Dictionary::gettext('queue_depth', :type => :column, :notfound => :titleize)
+      %th
+        = Dictionary::gettext('service_time_sec', :type => :column, :notfound => :titleize)
+      %th
+        = Dictionary::gettext('wait_time_sec', :type => :column, :notfound => :titleize)
+  %tbody
+    - @stats.each.sort_by(&:statistic_time).each do |stat|
+      %tr{:class => cycle('row0', 'row1')}
+        %td
+          = h(format_timezone(stat.statistic_time, Time.zone, "compare_hdr"))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.k_bytes_read_per_sec, :precision => 2))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.k_bytes_written_per_sec, :precision => 2))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.k_bytes_transferred_per_sec, :precision => 2))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.read_ios_per_sec, :precision => 2))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.read_hit_ios_per_sec, :precision => 2))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.write_ios_per_sec, :precision => 2))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.write_hit_ios_per_sec, :precision => 2))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.total_ios_per_sec, :precision => 2))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.avg_read_size, :precision => 2))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.avg_write_size, :precision => 2))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.pct_read, :precision => 2))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.pct_write, :precision => 2))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.pct_hit, :precision => 2))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.utilization, :precision => 2))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.response_time_sec, :precision => 2))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.queue_depth, :precision => 2))
+        %td{:align => "right"}
+          = h(number_with_precision(stat.service_time_sec, :precision => 2))
+        %td{:align => "right"}
```

Comment:
could we generate all this from some list of columns? ... and not repeat the same stuff over and over...

thx!